### PR TITLE
General code quality improvements

### PR DESCRIPTION
**Pull request by rsevilla87**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [x] Optimization
- [ ] Documentation Update

## Description

Some code improvements and reorganization

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: pkg/measurements/pod_latency.go [hunk 1]
```diff
@@ -169,7 +168,6 @@ func (p *podLatency) stop() error {
 		err = metrics.CheckThreshold(p.config.LatencyThresholds, p.latencyQuantiles)
 	}
 	if globalCfg.IndexerConfig.Type != "" {
-		log.Infof("Indexing pod latency data for job: %s", factory.jobConfig.Name)
```
Comment (vishnuchalla) [inline pkg/measurements/pod_latency.go:172]: Do we want to keep this log?

Comment (rsevilla87) [reply pkg/measurements/pod_latency.go:172]: Have moved it to the index() function

File: cmd/kube-burner/ocp.go [hunk 2]
```diff
@@ -38,44 +36,35 @@ func openShiftCmd() *cobra.Command {
...
-	esIndex := ocpCmd.PersistentFlags().String("es-index", "", "Elastic Search index")
-	metricsEndpoint := ocpCmd.PersistentFlags().String("metrics-endpoint", "", "YAML file with a list of metric endpoints")
-	alerting := ocpCmd.PersistentFlags().Bool("alerting", true, "Enable alerting")
-	uuid := ocpCmd.PersistentFlags().String("uuid", uid.NewV4().String(), "Benchmark UUID")
-	timeout := ocpCmd.PersistentFlags().Duration("timeout", 4*time.Hour, "Benchmark timeout")
-	qps := ocpCmd.PersistentFlags().Int("qps", 20, "QPS")
-	burst := ocpCmd.PersistentFlags().Int("burst", 20, "Burst")
-	gc := ocpCmd.PersistentFlags().Bool("gc", true, "Garbage collect created namespaces")
-	gcMetrics := ocpCmd.PersistentFlags().Bool("gc-metrics", false, "Collect metrics during garbage collection")
+	ocpCmd.PersistentFlags().StringVar(&workloadConfig.MetricsEndpoint, "metrics-endpoint", "", "YAML file with a list of metric endpoints")
```
Comment (vishnuchalla) [inline cmd/kube-burner/ocp.go:44]: why are we exposing the config attributes globally. I think we can just keep them as camel case like we have for our `kube-burner` command line as of now.

File: pkg/workloads/types.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	GcMetrics       bool
+	Indexer         indexers.IndexerType
+	Alerting        bool
+	Reporting       bool
+	Timeout         time.Duration
+	MetricsEndpoint string
+	ProfileType     string
+}
+
+type BenchmarkMetadata struct {
```
Comment (vishnuchalla) [inline pkg/workloads/types.go:50]: Same here about the global vars. I think we should only expose the required ones as globals.

File: cmd/kube-burner/ocp.go (hunk 2 above)
Comment (rsevilla87) [reply cmd/kube-burner/ocp.go:44]: It's needed to access a variable in a different package

**Pull request merged**